tests/unit/interactive/test_views.py

```diff
@@ -1,14 +1,24 @@
 from interactive.models import AnalysisRequest
 from interactive.views import AnalysisRequestCreate
+from jobserver.authorization import InteractiveReporter
 
-from ...factories import BackendFactory, ProjectFactory, WorkspaceFactory
+from ...factories import (
+    BackendFactory,
+    ProjectFactory,
+    ProjectMembershipFactory,
+    UserFactory,
+    WorkspaceFactory,
+)
 
 
-def test_analysisrequestcreate_get_success(rf, core_developer):
+def test_analysisrequestcreate_get_success(rf):
```

Comment:
I think you need some tests here to check that PermissionDenied is raised when someone doesn't have the correct role?

Reply:
Oops, yep 👍